apps/web/components/sections/profile/skeletons.tsx

```
@@ -0,0 +1,24 @@
+
+export function ProfileViewSkeleton() {
+  return (
+    <div className="space-y-6">
+      <div className="space-y-2">
+        <SkeletonText className="h-6 w-32" />
+        <SkeletonText className="h-4 w-96" />
+      </div>
+      <div className="grid grid-cols-1 gap-4 md:grid-cols-2 lg:grid-cols-3">
+        {Array.from({ length: 6 }).map((_, i) => (
+          <div
+            key={i}
```

Biome (2.4.15)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)